packages/voice/src/audio/AudioPlayer.ts

```diff
@@ -163,6 +164,19 @@ export type AudioPlayerEvents = {
 	) => Awaited<void>;
 };
 
+/**
+ * Stringifies an AudioPlayerState instance.
+ *
+ * @param state - The state to stringify
+ */
+function stringifyState(state: AudioPlayerState) {
+	return JSON.stringify({
+		...state,
+		resource: Reflect.has(state, 'resource'),
+		stepTimeout: Reflect.has(state, 'stepTimeout'),
+	});
```

Comment:
I just realised... aren't those Reflect.has useless? o.o

Reply:
Maybe, maybe not, but I'm not going to plan on doing big code changes in a linting PR.